subsys/bluetooth/host/gatt.c

```diff
@@ -5446,6 +5446,11 @@ int bt_gatt_subscribe(struct bt_conn *conn,
 		return -ENOTCONN;
 	}
 
+	if (params->value < BT_GATT_CCC_NOTIFY
+		|| params->value > BT_GATT_CCC_INDICATE) {
```

Comment:
`params->value` may also be `BT_GATT_CCC_NOTIFY | BT_GATT_CCC_INDICATE`.

```suggestion
	if (!IN_RANGE(params->value, BT_GATT_CCC_NOTIFY, BT_GATT_CCC_NOTIFY | BT_GATT_CCC_INDICATE) {
```

We could also consider defining a mask for this purpose